src/components/Tooltip.tsx

```diff
@@ -1,31 +1,89 @@
 import "./Tooltip.scss";
 
-import React from "react";
+import React, { useEffect } from "react";
+
+const getTooltipDiv = () => {
+  const existingDiv = document.querySelector<HTMLDivElement>(
+    ".ExcalidrawTooltip",
+  );
+  if (existingDiv) {
+    return existingDiv;
+  }
+  const div = document.createElement("div");
+  document.body.appendChild(div);
+  div.classList.add("ExcalidrawTooltip");
```

Comment:
```suggestion
  div.classList.add("excalidraw-tooltip");
```
So that CSS class names are consistent and we can follow BEM